### PR TITLE
Bscharenberg/wqxr on fastboot

### PR DESCRIPTION
**Pull request by bryn19**
go fast 

jira: https://jira.wnyc.org/browse/MT-147

### CODE REVIEW

File: app/components/site-chrome/template.hbs [hunk 1]
```diff
@@ -8,12 +8,12 @@
   <div class="sitechrome-top">
     <div class="sitechrome-topbody">
       <div class="sitechrome-topbody-right">
-          {{#if media.isLargeAndUp}}
+        <!-- if media is isLargeAndUp -->
```
Comment (kedmundson) [inline app/components/site-chrome/template.hbs]: Is this managed with a media query?

File: app/events/event/route.js [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
+import { reads } from '@ember/object/computed';
+import { inject as service } from '@ember/service';
 
 export default Route.extend({
+  fastboot: service(),
+  isFastBoot: reads('fastboot.isFastBoot'),
+
   titleToken(model) {
-    return model.get('title');
+    return get(model, 'title');
```
Comment (kedmundson) [inline app/events/event/route.js]: Out of curiosity, what's the benefit to this change?

File: app/events/event/route.js [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
   },
   model({url_path}) {
+    if (this.get('isFastBoot')) {
+      return {'title': ''}
```
Comment (kedmundson) [inline app/events/event/route.js]: At what point does this page title get populated?

File: app/routes/index.js [hunk 4]
```diff
@@ -11,22 +11,39 @@ export default Route.extend(PlayParamMixin, {
...
-    return RSVP.hash({
-      wqxrHome: this.store.findRecord('bucket', 'wqxr-home').then(b => {
-        return {
-          featuredItems: b.get('bucketItems').slice(0, 9),
-          otherItems: b.get('bucketItems').slice(9)
-        };
-      }),
-      wartopChunk: this.store.findRecord('chunk', 'wqxr-wartop-home').catch(()=>''),
-      membershipChunk: this.store.findRecord('chunk', 'wqxr-membership-home').catch(() => ''),
+    if (this.get('fastboot.isFastBoot')) {
```
Comment (kedmundson) [inline app/routes/index.js]: It'd be nice to DRY up this section a bit, maybe with an approach like this:

```
  let hash = {
    wqxrHome: this.store.findRecord('bucket', 'wqxr-home').then(b => {
      return {
        featuredItems: b.get('bucketItems').slice(0, 9),
        otherItems: b.get('bucketItems').slice(9)
      };
    }),  
  };

  if (!this.get('fastboot.isFastBoot')) {
    hash.wartopChunk = this.store.findRecord('chunk', 'wqxr-wartop-home').catch(()=>''),
    hash.membershipChunk = this.store.findRecord('chunk', 'wqxr-membership-home').catch(() => ''),
  }

  return RSVP.hash(hash);
```

File: app/styles/_site-chrome.scss [hunk 5]
```diff
@@ -6,6 +6,22 @@
   /* So the side nav will position correctly */
 }
 
+/* Hide certain elements based on window size. This must be done in css
+since */
```
Comment (kedmundson) [inline app/styles/_site-chrome.scss]: missing the end of this comment

File: cypress/fixtures/v1/comments/__story-list-comments.json [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (kedmundson) [inline cypress/fixtures/v1/comments/__story-list-comments.json]: Does Cypress auto-generate these fixtures?

File: app/components/site-chrome/template.hbs (hunk 1 above)
Comment (bryn19) [reply app/components/site-chrome/template.hbs]: whoops, forgot to take this out. yes, this is now managed w/ a media query in css.

File: app/events/event/route.js (hunk 2 above)
Comment (bryn19) [reply app/events/event/route.js]: i think it changed from object to an ember object, which made the change necessary.

File: app/events/event/route.js (hunk 3 above)
Comment (bryn19) [reply app/events/event/route.js]: on load when the route is rehydrated

File: app/routes/index.js (hunk 4 above)
Comment (bryn19) [reply app/routes/index.js]: good point, i'll do this!

File: app/styles/_site-chrome.scss (hunk 5 above)
Comment (bryn19) [reply app/styles/_site-chrome.scss]: idk about you but i want as much suspense and mystery in my comments as possible

File: cypress/fixtures/v1/comments/__story-list-comments.json (hunk 6 above)
Comment (bryn19) [reply cypress/fixtures/v1/comments/__story-list-comments.json]: nope. you gotta copy/paste

**Comment by bryn19** [comment]
@kedmundson pushed up a new version w/ changes

File: app/channel/route.js [hunk 7]
```diff
@@ -67,7 +68,7 @@ export default Route.extend(PlayParamMixin, {
   actions: {
     willTransition(transition) {
       if (get(this, "isFastBoot")) {
-        return;
+        //eturn;
```
Comment (kedmundson) [inline app/channel/route.js]: commented out return statement alert

**Pull request merged**